core-web/libs/dotcms-scss/angular/dotcms-theme/components/buttons/_button.scss

```diff
@@ -51,7 +51,8 @@
 }
 
 // Severity for basic button
-.p-button:enabled {
+.p-button:enabled,
+.p-button.p-element {
```

Comment:
What component have this `.p-element` selector?

Reply:
`.p-element` is a common class in all primeNG components, this is just for css specificity reasons